src/node/build/index.ts

```diff
@@ -767,8 +771,8 @@ function createEmitPlugin(
  * `outDir` in Vite config).
  */
 function getIndexHtmlOutputPath({ input, output }: Build, outDir: string) {
-  return input === 'index.html'
-    ? path.resolve(outDir, output.file || input)
+  return typeof input === 'string' && input.endsWith('.html')
```

Comment:
We should probably use `/\.(html|php)$/.test(input)` instead.

Reply:
Not sure for `php`.Can you explain that?

Reply:
@underfin PHP is just HTML with special `<?php` logic tags. These special tags are evaluated on the server before serving as plain HTML.